### ISSUE TITLE
Bug: message banners stack

### ISSUE THREAD

**Issue by Pinjasaur** [mattermost/focalboard#3694]
## Steps to reproduce the behavior

1. Start up a fresh version of e.g. personal server
2. Notice version & cloud banner stack

## Expected behavior

Not 100% sure, perhaps different colors or not stacking? Just felt odd to have them as shown in the screenshot.

## Screenshots (optional)

<img width="1512" alt="image" src="https://user-images.githubusercontent.com/6335792/184994807-ccd2d931-4764-4626-92b7-b202d70637a8.png">

## Edition and Platform

 - Edition: Personal Desktop / Personal Server / Mattermost Boards (plugin)
 - Version: main
 - Browser and OS: FF104 Mac

## Additional context (optional)

The cloud message is per-user (once dismissed, never comes back), and the version message is per-version (comes back next version) so this worst-case screenshot would be a brand new user.

**Comment by Pinjasaur** [comment]
This might be more of an enhancement, not a bug.

cc @asaadmahmood for UI/UX and @sbishel since I _believe_ you implemented both banners.

**Comment by asaadmahmood** [comment]
@wuwinson I don't think both banners should appear, the What's new banner should overlap the get your own free cloud, and when that is dismissed, we can show the get your own cloud server one (but even that not immediately, maybe on the next refresh or something?)

**Comment by wuwinson** [comment]
Agreed, the "What's New" banner should take precedence over the "Free cloud server" banner. Not sure how frequent a user refreshes the desktop app, but what if we delay the the cloud server banner until 24 hrs after the "What's New" banner is dismissed?

**Comment by wuwinson** [comment]
To clarify, this shouldn't be appearing on Personal Desktop, but the "What's New" banner should appear on Personal Server.
